library/rendering/registration/src/main/java/org/quiltmc/qsl/rendering/registration/impl/client/BlockRenderLayerMapImpl.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.quiltmc.qsl.block.extensions.impl.client;
+package org.quiltmc.qsl.rendering.registration.impl.client;
```

Comment:
idk how to feel about this

but if it's a move the copyright year still shouldn't be changed

Reply:
The license changes have been reverted.